tensorboard.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+open -a 'Google Chrome' http://localhost:6008/#scalars
```

Comment:
This command doesn't work on my OS (Ubuntu 16.04). Also I don't think everyone uses Chrome.